### PR TITLE
Add ServiceNow documentation

### PR DESCRIPTION
**Pull request by gsanchezu**
@rchakra1, @julianatienza, 👍 ?

### CODE REVIEW

File: Cloud Application Manager/Administering Your Organization/itsm-integrations.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+4. Once you have saved your configuration, you need to Authorize access for Cloud Application Manager to be able to talk to your ServiceNow instance by clicking the **Authorize** button:
+  ![ServiceNow CMDB authorize](../../images/cloud-application-manager/management/servicenow-cmdb-authorize.png)
+  When clicking on the **Authorize** button a ServiceNow screen will appear where you can **Allow** Cloud Application Manager to connect to your ServiceNow account.
+
+5. Once you have authorize the connection, you will see **Authorized** in your configuration, and you will have the option to **Reauthorize** in case you need to do so, or when your token expires:
+  ![ServiceNow CMDB authorized](../../images/cloud-application-manager/management/servicenow-cmdb-authorized.png)
+
+### Behavior
+
+Once ServiceNot CMDB integration has been configured, the following behavior will occur when deploying any instance through Cloud Application Manager.
```
Comment (julianatienza) [inline Cloud Application Manager/Administering Your Organization/itsm-integrations.md]: Please, change "Once ServiceNot" by "Once ServiceNow". Thanks!!!

Comment (julianatienza) [reply Cloud Application Manager/Administering Your Organization/itsm-integrations.md]: Please, change "behavior" word by "behaviour". Thanks!!!

**Comment by julianatienza** [comment]
@gsanchezu there is also an activity log when the ServiceNow ITSM CMDB integration is switched on/off. I'm not sure if this should be mentioned in the doc. Thanks

File: Cloud Application Manager/Administering Your Organization/itsm-integrations.md (hunk 1 above)
Comment (gsanchezu) [reply Cloud Application Manager/Administering Your Organization/itsm-integrations.md]: Thanks, @julianatienza, I addressed the first comment, but the second is expressed in American English, so it is ok as it is.
I also mentioned the Activity log.

**Comment by julianatienza** [comment]
ack

**Comment by lyzalou** [comment]
@rchakra1 and @gsanchezu is this good to go out?

**Comment by gsanchezu** [comment]
Yes @lyzalou, it can be published. Thanks!

**Pull request merged**